front/components/assistant_builder/actions/DustAppRunAction.tsx

```diff
@@ -18,34 +22,40 @@ export function isActionDustAppRunValid(
 
 export function ActionDustAppRun({
   owner,
-  actionConfigration,
+  action,
   updateAction,
   setEdited,
   dustApps,
 }: {
   owner: WorkspaceType;
-  actionConfigration: AssistantBuilderDustAppConfiguration | null;
-  updateAction: (
-    setNewAction: (
-      previousAction: AssistantBuilderDustAppConfiguration
-    ) => AssistantBuilderDustAppConfiguration
-  ) => void;
+  action: AssistantBuilderActionConfiguration;
```

Comment:
I'm not sure I really understand why we broaden the type to any type of action config ? this component should always be getting a Dust App run config no ?

Reply:
We need to pass the name and description of the action to be able to change them when we pick an app. Do you see another cleaner way to do it?

Reply:
ah yes I see....